### ISSUE TITLE
How to change the background image for the header

### ISSUE THREAD

**Issue by nisrulz** [zzossig/hugo-theme-zzo#49]
I have already tried placing `background.jpg` under `static/images/header/`

**Comment by zzossig** [comment]
Did you set param at config/_default/params.toml ?
```toml
...
homeHeaderType = "img"
...
```

**Comment by nisrulz** [comment]
Yes. PFA the screenshots.

![Screenshot from 2019-12-30 14-04-54](https://user-images.githubusercontent.com/2096087/71583147-82f6c100-2b0d-11ea-8941-44234f09cf5f.png)
![Screenshot from 2019-12-30 14-05-35](https://user-images.githubusercontent.com/2096087/71583148-84c08480-2b0d-11ea-929e-31163f645c36.png)

**Comment by zzossig** [comment]
Then, did you make a _index.md file at content folder?
![주석 2019-12-30 220835](https://user-images.githubusercontent.com/52706977/71583244-eeae4d80-2b50-11ea-8034-4148ef34f6fc.png)

**Comment by nisrulz** [comment]
Yes, sire :smiley_cat: 

![Screenshot from 2019-12-30 14-09-18](https://user-images.githubusercontent.com/2096087/71583279-087a7100-2b0e-11ea-9d79-cc394ce2ee9a.png)

**Comment by zzossig** [comment]
Then, it must work if you have a file at root/static/images/header/background.jpg file

**Comment by nisrulz** [comment]
Exactly. That is what I shared above. It just doesn't work. :man_shrugging:

**Comment by nisrulz** [comment]
An update:
Moving `background.jpg` outside of `header` folder into its parent `images` and updating the link in `_index.en.md` file:

``` imageSrc: images/background.jpg```

Fixed the issue. I can now load the custom background image.

Note: I tested with the image being in `header` folder vs its parent `images` and it only works when it is inside `images` and not when it is placed inside `header` child folder.

**Comment by zzossig** [comment]
This is very strange. 
When I tested it on Window10, it works well, But
When I tested it on Ubuntu, the Hugo is not following the lookup order.
Hugo must find a file at root/static image first. If the file does not exist, then Hugo searches the file at the theme folder, which is not the case on Ubuntu.
I think I have to open a new issue on the Hugo repo regarding this.
I'm closing this issue now.

The quick solution: just change the filename or path, and you are good to go
